bitnami/logstash/templates/headless-svc.yaml

```
@@ -25,5 +25,10 @@ spec:
   type: ClusterIP
   clusterIP: None
   ports:
-    {{- include "common.tplvalues.render" (dict "value" .Values.service.ports "context" $) | nindent 4 }}
+  {{- range $service := .Values.service.ports }}
+    - name: {{ $service.name }}
+      port: {{ $service.port }}
+      targetPort: {{ $service.targetPort }}
+      protocol: {{ $service.protocol }}
+  {{- end }}
```

This should be compatible with using templates.
```suggestion
    {{- range $port := .Values.service.ports }}
    {{- omit (include "common.tplvalues.render" ( dict "value" $port "context" $ ) | fromYaml) "nodePort" | toYaml | nindent 4 }}
    {{- end }}
```